Godeps/_workspace/src/github.com/openshift/openshift-sdn/pkg/ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -41,10 +41,12 @@ function docker_network_config() {
 
 DOCKER_NETWORK_OPTIONS='${DOCKER_NETWORK_OPTIONS}'
 EOF
-
-		systemctl daemon-reload
-		systemctl restart docker.service
-
+		## linux bridge
+		ip link set lbr0 down || true
+		brctl delbr lbr0 || true
+		brctl addbr lbr0
+		ip addr add ${local_subnet_gateway}/${local_subnet_mask_len} dev lbr0
+		ip link set lbr0 up
```

Comment:
@danwinship @rajatchopra Can you review these? Didn't we just bump os-sdn for this stuff yesterday?